src/main/java/uk/gov/hmcts/darts/event/exception/EventError.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.darts.event.exception;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.darts.common.exception.DartsApiError;
+
+@Getter
+@RequiredArgsConstructor
+public enum EventError implements DartsApiError {
+
+    EVENT_DATA_NOT_FOUND(
+          "422",
+          HttpStatus.BAD_REQUEST,
+          "Data on the event could not be reconciled with Darts records"),
+
+    EVENT_HANDLER_NOT_FOUND("422",
```

Comment:
Please check this with @chrisbellingham-hmcts  - I'm pretty sure he means for these to be a distinct numeric within an error type, if you look at his example `AuthenticationError`. See what he says about the status code too 👍

Reply:
Yeah think you must be right about the errorTypeNumeric